features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/db2.sql

```diff
@@ -1533,7 +1533,7 @@ IDN_CONFIG_RESOURCE (ID) ON DELETE CASCADE ON UPDATE RESTRICT
 /
 
 CREATE TABLE IDN_REMOTE_FETCH_CONFIG(
-    ID INTEGER NOT NULL,
+	ID VARCHAR(255) NOT NULL,
```

Comment:
Minor indentation issue here

Reply:
Fixed in https://github.com/wso2/carbon-identity-framework/pull/3143